sources/web/datalab/polymer/modules/GapiManager.ts

```diff
@@ -34,7 +40,8 @@ class GapiManager {
     // Loads the gapi client library and the auth2 library together for efficiency.
     // Loading the auth2 library is optional here since `gapi.client.init` function will load
     // it if not already loaded. Loading it upfront can save one network request.
-    GapiManager._loadClientId()
+    console.log('== loading gapi');
+    return GapiManager._loadClientId()
```

Comment:
Are you proposing that all functions that return a Promise should have a name that ends in Async? I don't think I agree with that (and we have a ton of "Async"s to add if we're going to do it). It sounds kind of like hungarian notation.
Or am I misunderstanding your request?